ckan/logic/auth/get.py

```diff
@@ -178,14 +195,19 @@ def vocabulary_show(context, data_dict):
     # Allow viewing of vocabs by default
     return {'success': True}
 
+
 def tag_show(context, data_dict):
     # No authz check in the logic function
     return {'success': True}
 
+
 def user_show(context, data_dict):
     # By default, user details can be read by anyone, but some properties like
     # the API key are stripped at the action level if not not logged in.
-    return {'success': True}
+    if not asbool(config.get('ckan.auth.public_user_details', True)):
+        return restrict_anon(context)
+    else:
+        return {'success': True}
 
```

Comment:
Do we have any 403 handlers for flask application? After disabling public_users, visiting '/user' page results in base **Forbidden** page. I just don't remember, whether it's ok and we are going to implement error handlers separately or they already are implemented

Reply:
lets discuss at the meeting later